aggregator/src/util.rs

```diff
@@ -99,12 +99,16 @@ pub(crate) fn get_indices(preimages: &[Vec<u8>]) -> (Vec<usize>, Vec<usize>) {
 #[inline]
 // assert two cells have same value
 // (NOT constraining equality in circuit)
-pub(crate) fn assert_equal<F: Field>(a: &AssignedCell<F, F>, b: &AssignedCell<F, F>) {
+pub(crate) fn assert_equal<F: Field>(
+    a: &AssignedCell<F, F>,
+    b: &AssignedCell<F, F>,
+    description: &str,
+) {
     let mut t1 = F::default();
     let mut t2 = F::default();
     a.value().map(|f| t1 = *f);
     b.value().map(|f| t2 = *f);
-    assert_eq!(t1, t2)
+    assert_eq!(t1, t2, "{description}",)
```

Comment:
does this work?    I am not sure, i usually will write assert_eq!(t1, t2, format!("{description}"));

Reply:
from clippy:
```
   --> aggregator/src/util.rs:130:9
    |
130 |         assert_eq!(t1, t2, "{}",description)
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#uninlined_format_args
help: change this to
    |
130 -         assert_eq!(t1, t2, "{}",description)
130 +         assert_eq!(t1, t2, "{description}")
```